zfit/models/physics.py

```diff
@@ -576,6 +576,101 @@ def generalized_gaussexptail_func(x, mu, sigmal, alphal, nl, sigmar, alphar, nr)
     )
 
 
+def gaussexptail_integral(limits, params, model):
+    del model
+    mu = params["mu"]
+    sigma = params["sigma"]
+    alpha = params["alpha"]
+    n = params["n"]
+
+    lower, upper = limits._rect_limits_tf
+
+    return gaussexptail_integral_func(mu, sigma, alpha, n, lower, upper)
+
+
+@z.function(wraps="tensor", keepalive=True)
+def gaussexptail_integral_func(mu, sigma, alpha, n, lower, upper):
+    sqrt_pi_over_two = np.sqrt(np.pi / 2)
+    sqrt2 = np.sqrt(2)
+
+    abs_sigma = znp.abs(sigma)
+    abs_alpha = znp.abs(alpha)
+    tmin = (lower - mu) / abs_sigma
+    tmax = (upper - mu) / abs_sigma
+
+    alpha_negative = tf.less(alpha, 0)
+    # do not move on two lines, logic will fail...
+    tmax, tmin = (
+        znp.where(alpha_negative, -tmin, tmax),
+        znp.where(alpha_negative, -tmax, tmin),
+    )
+
+    guass_entire_integral = abs_sigma * sqrt_pi_over_two * (tf.math.erf(tmax / sqrt2) - tf.math.erf(tmin / sqrt2))
+    exp_entire_integral = (
+        abs_sigma / n * znp.exp(-0.5 * znp.square(abs_alpha) + n * abs_alpha) * (znp.exp(n * tmax) - znp.exp(n * tmin))
+    )
+    gauss_integral_alpha_tmax = (
+        abs_sigma * sqrt_pi_over_two * (tf.math.erf(tmax / sqrt2) - tf.math.erf(-abs_alpha / sqrt2))
+    )
+    exp_integral_alpha_tmin = (
+        abs_sigma
+        / n
+        * znp.exp(-0.5 * znp.square(abs_alpha) + n * abs_alpha)
+        * (znp.exp(-n * abs_alpha) - znp.exp(n * tmin))
```

Comment:
combine the exp terms?

Reply:
Yes, I will merge/combine stuff all over the place as soon as I'm sure it's correct.